bin/dev-beetle

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euxo pipefail
+
+SERVICE=beetle-shell
+
+docker-compose -f docker-dev/docker-compose.yml up --no-recreate -d ${SERVICE}
+
+COMMAND=${@:-bash}
+echo Run \'$COMMAND\' in container context
+
+docker exec -ti ${SERVICE} ${COMMAND}
+
```

Comment:
What's the use of this? If this this were to be kept, it would need to move to the script folder.